app/services/end_of_cycle_timetable.rb

```diff
@@ -180,7 +193,7 @@ def self.can_add_course_choice?(application_form)
   end
 
   def self.can_submit?(application_form)
-    RecruitmentCycle.current_year == application_form.recruitment_cycle_year
+    current_year == application_form.recruitment_cycle_year
```

Comment:
We've noticed the duplication here between this line and line 184 - there are plans afoot to review the class methods in a future PR (and their callers) which will resolve. Else for now we can delegate this method to `.current_cycle?` if that is preferred